### DISCUSSION TITLE
Sending commands to the System Bridge via MQTT

### DISCUSSION THREAD

**Discussion by arunoruto** [timmo001/system-bridge#790] [Q&A] [answered]
So I have connected the system-bridge with the mosquitto broker on my home assistant. I am able to read the data received by the bridge, but I am not able to send commands to it. Is this intentionally ignored or just missing in the implementation? 
If it has been implemented, I wasn't able to find any documentation and I tried replicating a call similar to the REST API, but nothing happens.

**Comment by timmo001** [comment, marked as answer]
MQTT just receives data at the minute. Feel free to open a feature request for what you would like to use via MQTT.

Here are the the docs for MQTT: https://system-bridge.timmo.dev/docs/mqtt
